packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -1074,6 +1147,35 @@ export class Service extends cdk.Resource implements iam.IGrantable {
       throw new Error('configurationValues cannot be provided if the ConfigurationSource is Repository');
     }
 
+    if (this.props.healthCheckConfiguration?.path !== undefined) {
+      if (this.props.healthCheckConfiguration?.protocol !== HealthCheckProtocolType.HTTP) {
+        throw new Error('path is only applicable when you set Protocol to HTTP');
+      }
+      if (this.props.healthCheckConfiguration?.path.length === 0) {
+        throw new Error('path length must be greater than 0');
+      }
+    }
+    if (this.props.healthCheckConfiguration?.healthyThreshold !== undefined) {
+      if (this.props.healthCheckConfiguration?.healthyThreshold < 1 || this.props.healthCheckConfiguration?.healthyThreshold > 20) {
+        throw new Error('healthyThreshold must be greater than or equal to 1 and less than or equal to 20');
```

Comment:
We should add the received value in the error message as a feedback to the user and make the message more concise:
```
throw new Error(`healthyThreshold must be between 1 and 20, got ${healthCheckConfiguration.healthyThreshold}`);
```